cmd/backend/main.go

```diff
@@ -79,12 +80,12 @@ func main() {
 	healthyLevel := 25
 	weakLevel := 5 // larger than maxCriticalDay
 	maxDayCritical := 3
-	foodReqStrong := 20.0
-	foodReqHealthy := 15.0
-	foodReqWeak := 10.0
+	tauStrong := 20.0
+	tauHealthy := 15.0
+	tauWeak := 10.0
```

Comment:
I think I preferred the old names for tauStrong, tauHealthy, tauWeak haha. (this is minor and maybe can just make some comments to explain those variables).

Reply:
hahahh we have some diverging opinion on that. On a theoretical perspective, `tau` makes more sense as this is a characteristic of the curve/function we model. However, on a practical level, `tau` is defined as "the amount of food needed to stay in a given HP level _if you wake up on the lower border of this level_". From this last "if" condition, it is possible for an agent to _eat less than_ `tau` food and still wake up in the same level the following day (if he wakes up with _more_ than just the level's lower border HP value. Therefore, I think `foodReq` can be misleading

Reply:
yeah makes sense thanks!